lib/modulesync.rb

```diff
@@ -50,8 +58,16 @@ def self.module_files(local_files, path)
     local_files.map { |file| file.sub(/#{path}/, '') }
   end
 
-  def self.managed_modules(config_file, filter, negative_filter)
-    managed_modules = Util.parse_config(config_file)
+  def self.managed_modules(config_file, filter, negative_filter, topic)
+    if topic
+      managed_modules = []
+      GITHUB.org_repos(GITHUB_ORGANIZATION).each do |repo|
```

Comment:
I'm wondering if `managed_modules = GITHUB.org_repos(GITHUB_ORGANIZATION).filter { |r| r.topics.include?(topic) }.map(&:name)` is easier to read